extensions/classify/testdata/operation.json

```diff
@@ -0,0 +1,37 @@
+{
+  "phase": "LOG",
+  "dimensions": [
+    {
+      "name": "operation",
+      "values": [
+        {
+          "value": "GetReview",
+          "match": {
+            "uri": {
+              "regex": "^/reviews/\d*"
+            },
+            "method": "GET"
+          }
+        },
+        {
+          "value": "ListReviews",
+          "match": {
+            "uri": {
+              "exact": "/reviews/"
```

Comment:
in testing this: we should add a testcase for something like `/books/{id}/chapter/{id}`